### ISSUE TITLE
Error on Verify:  'SerialGpsPrint' declared void

### ISSUE THREAD

**Issue by gilgain** [HexAirbot/HexNanoMWC_HEX#9]
This is the error log when I verify sketch:

Arduino: 1.6.5 (Windows 7), Board: "Arduino Leonardo"

HexNanoMWC_HEX:68: error: variable or field 'SerialGpsPrint' declared void
HexNanoMWC_HEX:68: error: 'prog_char' was not declared in this scope
HexNanoMWC_HEX:68: error: 'str' was not declared in this scope
variable or field 'SerialGpsPrint' declared void

  This report would have more information with
  "Show verbose output during compilation"
  enabled in File > Preferences.

**Comment by NjerOeter** [comment]
I´m not sure if you are still interested or if you already have found a solution, but anyways, you just need to switch the "prog_char" in the GPS file with only "char" and it compiles again.

**Comment by gilgain** [comment]
Yes, I'm still very interested...just life got busy and I needed time. So with this fix suggested, will it compile on current version of Arduino IDE, or do I still need to install older version?

I look forward to using it again soon...after life slows down some for me. 

Thanks NjerOeter.

**Comment by NjerOeter** [comment]
I actually haven´t tried to push it to my Flexbot yet, but compiling worked for me with the newest version.
